Orange/widgets/data/owcolor.py

```diff
@@ -275,14 +275,10 @@ def change_color(self, index):
 
 
 class OWColor(widget.OWWidget):
-    """
-    Set color legend for variables
-
-    Widget for defining specific color palettes for discrete and continuous
-    features
-    """
-
     name = "Color"
+    description = "Set color legend for variables."
+    long_description = "Widget for defining specific color palettes " \
```

Comment:
Not after this PR. I removed long_description from display as it sometimes fell out of bounds and it also somehow took docstrings with it (I don't know where that was set). I discussed it with @janezd and we decided to keep only short descriptions. If you have better suggestions, you're welcome.